GA Prompt Execution Pages (#372)
vellum-ai/vellum-client-generator: fern/docs/content/change-log/2024-08.mdx

title: Changelog | August, 2024
---

## Prompt Execution Pages

_August 22nd, 2024_

> If you wanted to drill into a single prompt, previously your only recourse was to navigate to the Prompt Deployment's executions table and try to filter for the id

Reviewer: ```suggestion
If you wanted to drill into a single Prompt Execution, previously you’d have to navigate to the Prompt Deployment's Executions table and try to filter for the specific Execution ID
```

> you're looking for. Now each row has a navigable link accessible on the table: 

Reviewer: ```suggestion
you're looking for. Now each row has a navigable link accessible from the table: 
```

![Prompt Execution Link](https://storage.googleapis.com/vellum-public/help-docs/changelogs/2024-08/prompt-execution-link.png)

> This will navigate you to a dedicated page representing thie Prompt's execution. From here, you could inspect the raw HTTP data sent to and from the provider,

Reviewer: ```suggestion
This will navigate you to a dedicated page representing that specific Prompt Execution. From here, you can see details about the Execution like the raw HTTP data sent to and from the provider,
```
